src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -376,8 +386,9 @@
                     (catch Exception e
                       (log/warn e
                                 (i18n/tru "An exception occurred while reading the metadata file for the ''{0}'' task"
-                                          (str module-name "::" task-name)))))]
+                                          (str module-name "::" task-name )))))]
```

Comment:
Why wouldn't this error message also use the `full-task-name` function?

Reply:
The `full-task-name` function will omit the task part of the name for the init task, so `exec::init` will show up as `exec`. To aid in debugging, we felt it would be best to always have both components.